core/src/main/java/com/netflix/conductor/core/utils/JsonUtils.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.conductor.core.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Map;
+
+@Singleton
+public class JsonUtils {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public JsonUtils() {
+    }
+
+    private Object getJson(String jsonAsString) {
+        try {
+            return objectMapper.readValue(jsonAsString, Object.class);
+        } catch (Exception e) {
+            return jsonAsString;
```

Comment:
Exception is swallowed here, consider propagating the exception and logger statements. If no exception is propagated add doc to this method or the calling method for the reasoning behind not propagating the exception.

Reply:
Added a log message and added this to the javadoc for this method.